packages/mobile/src/send/utils.ts

```diff
@@ -93,13 +96,29 @@ function dailySpent(now: number, recentPayments: PaymentInfo[]) {
   return amount
 }
 
-export function isPaymentLimitReached(
-  now: number,
-  recentPayments: PaymentInfo[],
-  initial: number
+export function isPaymentLimitReached(amountToTransfer: BigNumber): boolean {
+  return _isPaymentLimitReachedWithStore(store.getState(), amountToTransfer)
```

Comment:
Nit: I think it's cleaner to have `store.getState` called within the inner function rather than here as a function argument. The function name `_isPaymentLimitReachedWithStore` implies that we are checking the store and there realistically isn't ever another value we'd want to use instead as an argument

Reply:
It's true it may lead to confusion, I use it from the test and it's easier to call it with a mock state, so I just renamed the function to say `state` instead of `store`, hope that solves the concern.

Reply:
Writing here for other reviewers, I ended up changing this code to be a custom hook instead `useDailyTransferLimitValidator`, I think it's cleaner this way :)